statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/httpfn/HttpFunction.java

```diff
@@ -221,11 +229,14 @@ private InvocationResponse unpackInvocationResultOrThrow(
           "Failure forwarding a message to a remote function " + self, asyncResult.throwable());
     }
     InputStream httpResponseBody = responseBody(asyncResult.value());
-    FromFunction fromFunction = parseProtobufOrThrow(FromFunction.parser(), httpResponseBody);
-    checkState(
-        fromFunction.hasInvocationResult(),
-        "The received HTTP payload does not contain an InvocationResult, but rather [%s]",
-        fromFunction);
-    return fromFunction.getInvocationResult();
+    try {
```

Comment:
@igalshilman The changes here seem unrelated to the ticket. Would be nice if this is at least a separate commit, as it is a bit confusing at first glance to see an unrelated change.
Just a comment for the future, no need to change this now, I'll proceed to merge as is.

Also as a side comment: this change is to make things easier for us to test the WIP remote http function, right? If yes, I think we deserve a JIRA to keep track of this.